SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -56,6 +56,9 @@ struct ConfigureConnection
 		Params.network.modular_kcp.upstream_heartbeat = &HeartbeatParams;
 #endif
 
+		Params.network.modular_kcp.security_type = GetDefault<USpatialGDKSettings>()->bUseSecureConnection ? WORKER_NETWORK_SECURITY_TYPE_TLS : WORKER_NETWORK_SECURITY_TYPE_INSECURE;
+		Params.network.modular_tcp.security_type = GetDefault<USpatialGDKSettings>()->bUseSecureConnection ? WORKER_NETWORK_SECURITY_TYPE_TLS : WORKER_NETWORK_SECURITY_TYPE_INSECURE;
```

Comment:
TCP is only used for server to runtime - would we ever want that on?

Reply:
If we do, we should probably be able to configure them separately (ie. 2 spatial flags)

Reply:
Hmmm, but what if for a debugging purpose a developer wants to use TCP instead of KCP for client connections? I'd say they'd still want the option. Disabling encryption for server to runtime always actually makes sense though. Think we should do a check for UnrealWorker and disable encryption, then have a single setting for either TCP / KCP for enabling encryption on clients. Sound good?

Reply:
Wouldn't just having the option to customise both client and server give us all the flexibility we need?